src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -257,6 +257,9 @@ internal sealed partial class ConsoleHost
                         s_theConsoleHost.UI.WriteWarningLine(preStartWarning);
                     }
 
+                    // Send startup telemetry for ConsoleHost startup
+                    ApplicationInsightsTelemetry.SendPSCoreStartupTelemetry();
```

Comment:
@lzybkr initially I was also calculating a checksum of the SMA assembly which was removed, and now there's less than 2ms increase when sending telemetry. (see comments below).